components/table/table-wrapper.js

```diff
@@ -538,8 +541,9 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 
 		// observes mutations to <table>'s direct children and also
 		// its subtree (rows or cells added/removed to any descendant)
-		this._registerMutationObserver('_tableMutationObserver', this._handleTableChange.bind(this), this._table, {
-			attributes: true, /* required for legacy-Edge, otherwise attributeFilter throws a syntax error */
```

Comment:
Seemed like a good time to clean up this setting that was only there for legacy-Edge.